websites/F/FormulaOneTv/presence.ts

```diff
@@ -0,0 +1,79 @@
+const presence = new Presence({ clientId: "916438450952097834" }),
+  strings = presence.getStrings({
+    play: "presence.playback.playing",
+    pause: "presence.playback.paused",
+    browsing: "general.browsing"
+  });
+
+function getEpochInMs(): number {
+  return Math.floor(+new Date() / 1000);
+}
+
+function parseTimeToMilliseconds(length: string): number {
+  const [seconds, minutes, hours] = length
+    .split(":")
+    .reverse()
+    .map((val) => parseInt(val, 10));
+
+  return seconds + (minutes || 0) * 60 + (hours || 0) * 3600;
+}
+
+const presenceData: PresenceData = {
+  largeImageKey: "logo_512"
+};
+
+async function setWatchingVideoActivity() {
+  const titleText = document.querySelector(".media-body h5")?.textContent;
```

Comment:
Don’t declare a variable when it is only used once

Reply:
It's used once for validating that it can get the title and twice for assigning the actual title.
Would this be better instead?
```
presenceData.details = `Watching ${document.querySelector(".media-body h5")?.textContent || "a race" }`
```
I feel like it's much worse for readability.

Reply:
We are disallowing one time vars in this PR - #4946

Reply:
Fixed